packages/unified-pipeline/src/components/Canvas/elements/Nodes/AtomicNode/AtomicNode.tsx

```diff
@@ -105,11 +105,11 @@ export default function AtomicNode({ isConnectable, data, id, xPos, yPos, zIndex
           <div
             style={{
               width,
-              height
+              minHeight: height
             }}
-            className="content-layer p-2.5 rounded-md">
+            className="content-layer p-2.5 rounded-md space-y-2.5">
             {icon}
-            <Text className="text-[11px] text-white font-normal leading-4">{name}</Text>
+            <Text className="text-[11px] text-white font-normal leading-4 line-clamp-2">{name}</Text>
```

Comment:
Once this change is reviewed, we can create a tailwind component class for node styles and it can be added for all node types.